### PR TITLE
Fixes version info displaying in the webui

### LINKED ISSUE #242: Version is not set by Makefile build target
When using the Thanos UI, looking at the runtime and build information produces this error in the logs:

```
level=warn name=query-1 ts=2018-03-08T13:19:32.532399482Z caller=ui.go:147 msg="template expansion failed" err="template: :80:24: executing \"content\" at <.Version.Version>: can't evaluate field Version in type string"
```

The version should be set when building the binary.

### PR DESCRIPTION
**Pull request by alvaroaleman**
Resolves #242

### CODE REVIEW

File: pkg/query/ui/ui.go [hunk 1]
```diff
@@ -70,11 +82,18 @@ func (u *UI) status(w http.ResponseWriter, r *http.Request) {
 	u.executeTemplate(w, "status.html", struct {
 		Birth   time.Time
 		CWD     string
-		Version string
+		Version PrometheusVersion
```
Comment (Bplotka) [inline pkg/query/ui/ui.go]: I think when building with promu (`make build` do that) we fill `-X {{repoPath}}/vendor/github.com/prometheus/common/version.Version={{.Version}}` and things like that (see `.promu.yaml`)

Cannot we use that instead here?

Comment (alvaroaleman) [reply pkg/query/ui/ui.go]: Well the root cause is that the template expects `Version` to be a struct and not a string: https://github.com/prometheus/prometheus/blob/master/web/ui/templates/status.html#L56

The info from `github.com/prometheus/common/version` is used to fill that struct, see line 90-95

Comment (Bplotka) [reply pkg/query/ui/ui.go]: ah.. yea, but we are in control of this template, so we can change it as we want. But, sorry I see now what's the issue. 

All good, but let's just rename it to `ThanosVersion` maybe?

**Comment by Bplotka** [comment]
So yea, let's merge this PR (maybe with `thanosVersion` named struct), and we can talk if this page should be even there, later

File: pkg/query/ui/ui.go (hunk 1 above)
Comment (alvaroaleman) [reply pkg/query/ui/ui.go]: Updated that

**Comment by Bplotka** [comment]
Thanks!

**Pull request merged**